x-pack/plugins/maps/public/connected_components/toolbar_overlay/feature_draw_controls/feature_edit_tools/feature_edit_tools.tsx

```diff
@@ -66,6 +67,7 @@ export function FeatureEditTools(props: Props) {
               aria-pressed={drawLineSelected}
               isSelected={drawLineSelected}
               display={drawLineSelected ? 'fill' : 'empty'}
+              isDisabled={isWaiting}
```

Comment:
Nit: I think it would be nice to have a tooltip explaining why the button is disabled. We could use a conditional tooltip wrapper like https://github.com/elastic/kibana/blob/main/x-pack/packages/kbn-random-sampling/src/ui/slider_control/index.tsx#L40-L45.

In most cases the user might not even notice that the button is disabled since the default `refresh_interval` on an index is `1s`. But if an index is configured for a higher `refresh_interval` a user might wonder why the button is disabled. Might be nice to tell the user that the data is being written and possibly suggest setting a lower `refresh_interval` on the index.

Reply:
I looked into this and there are 2 complications
1) wrapping EuiButton in EuiTooltip removes cursor css style. This can be resolved with some css so not a huge deal
2) tooltip messaging. I don't think we should leak implementation details like index refresh_interval to users. Changing this value has cost implications and would be rather complex to explain.

I think since feature editing is still beta, we can just leave as is and see what kind of user feed back we get.